### PR TITLE
Remove App component controlling resizing

### PR DESCRIPTION
**Pull request by armenzg**
I've tried using the app without it and I can't notice any issues.

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -13,59 +13,21 @@ import AndroidV2 from './views/AndroidV2';
...
-            <Route path='/' exact component={Home} />
-            <Route path='/crashes' exact component={ReleaseCrashes} />
-            <Route path='/crashes/beta' component={BetaCrashes} />
-            <Route path='/status' component={Status} />
-            <QuantumRoutes />
-            <Route component={NoMatch} />
-          </Switch>
-        </App>
+        <Switch>
+          <Route path='/android' exact component={Android} />
```
Comment (eliperelman) [inline src/routes.js]: The idiomatic way to quote props is double quotes.

File: src/routes.js [hunk 2]
```diff
@@ -13,59 +13,21 @@ import AndroidV2 from './views/AndroidV2';
...
+          <Route path='/android/v2' exact component={AndroidV2} />
+          <Route path='/' exact component={Home} />
```
Comment (eliperelman) [inline src/routes.js]: You can avoid the usage of the `exact` prop if you rearrange these from most-specific to least specific:

```jsx
<Route path="/android/v2" component={AndroidV2} />
<Route path="/android" component={Android} />
<Route path="/crashes/beta" component={BetaCrashes} />
<Route path="/crashes" component={ReleaseCrashes} />
<Route path="/status" component={Status} />
<Route path="/devtools" component={Devtools} />
<QuantumRoutes />
<Route path="/" component={Home} />
<Route component={NoMatch} />
```

File: src/routes.js [hunk 3]
```diff
@@ -13,59 +13,21 @@ import AndroidV2 from './views/AndroidV2';
...
+          <Route path='/crashes' exact component={ReleaseCrashes} />
+          <Route path='/crashes/beta' component={BetaCrashes} />
+          <Route path='/status' component={Status} />
+          <Route path='/devtools' component={Devtools} />
+          <QuantumRoutes />
```
Comment (eliperelman) [inline src/routes.js]: Why do these routes exist in a different file?

Comment (armenzg) [reply src/routes.js]: It pre-dates me.

**Comment by armenzg** [comment]
I've addressed all you pointed out. Thanks!

**Comment by armenzg** [comment]
Thank you so much!

**Pull request merged**